js/signin.js

```diff
@@ -0,0 +1,88 @@
+const emailInput = document.querySelector("#email");
+const pwInput = document.querySelector("#password");
+const pwWrapper = document.querySelector(".password-wrapper");
+const signinBtn = document.querySelector("#login-button");
+const eyeBtn = document.querySelector(".eye-off");
+
+const emailEmptyMsg = document.createElement("span");
+const emailInvalidMsg = document.createElement("span");
+const pwEmptyMsg = document.createElement("span");
+
+let regex = new RegExp("[a-z0-9]+@[a-z]+.[a-z]{2,3}");
+
```

Comment:
`어떤 것을 판별하기 위한` 정규식인지를 변수명에서 나타내주는게 좋습니다.
그리고 이렇게 선언된 값은 재할당 될 이유가 없기 때문에 `const`로 선언해주는 것을 추천드려요!

정규식 생성자를 사용한 것도 좋은 시도네요 👍
정규식은 리터럴로 선언할 때, `/`로 시작-종료되면 자동으로 정규식으로 해석하니 참고해주셔도 좋을 것 같아요!

```suggestion

const REGEX_EMAIL = new RegExp("[a-z0-9]+@[a-z]+.[a-z]{2,3}");
```